flint.ui/src/views/flint/ConfigurationsRothC.vue

```diff
@@ -153,6 +154,12 @@ export default {
       }
     })
 
+    if (store.state.point.firstRun === true) {
+      // Then return early. This also makes sure that user doesn't get
+      // `Modal.confirm` prompt if it's the first time.
+      store.commit('setRunStatus', false)
+    }
+
```

Comment:
Since the bug is present in RothC, you should target `store.state.rothc.firstRun` and not the `point.firstRun`, which would be used for Point simulation.

You should add a new store variable called `firstRun` in `src/store/modules/rothc.js` and write a mutation for it first. Refer to my work in `point.js`.

https://github.com/moja-global/FLINT-UI/blob/f852295cf162342d80ed3fac27a4af3cf8bc1961/flint.ui/src/store/modules/point.js#L149-L155

Reply:
Done with the changes, can you please check now?
Thanks for instructing me :)